InvenTree/InvenTree/tasks.py

```diff
@@ -447,3 +454,74 @@ def send_email(subject, body, recipients, from_email=None, html_message=None):
         fail_silently=False,
         html_message=html_message
     )
+
+
+@scheduled_task(ScheduledTask.DAILY)
+def check_for_migrations(worker: bool = True):
+    """Checks if migrations are needed.
+
+    If the setting auto_update is enabled we will start updateing.
+    """
+    # Test if auto-updates are enabled
+    if not get_setting('INVENTREE_AUTO_UPDATE', 'auto_update'):
```

Comment:
@eeintech maybe this setting is of so its not working?

Reply:
If it is required, I don't find it in the "Server Settings" of the UI

Reply:
I would say its required to set this via env `export INVENTREE_AUTO_UPDATE=True` or via config key `auto_update`, but @matmair could tell more.

Reply:
It is intentionally not exposed as I am worried about crashes if it is changed while running.